UMA-Bots/Receiving-Chain-Relay-Dispute/README.md

```diff
@@ -0,0 +1,36 @@
+# Relay Dispute Detection bot
+
+## Description
+
+This bot detects relay disputes on the receiving chains for the  [Across v2 Protocol](https://across.to/) - a multichain bridge which uses [UMA](https://umaproject.org/) as its source of on-chain data and validation. For more details refer [here](https://discourse.umaproject.org/t/forta-monitors-across-v2-request-for-proposals/1569).
+
+## Supported Chains
+- Mainnet
+  
+## Alerts
+
+- UMA-3
+  - Fired when a dispute occurs on the receiving chain
+  - Severity is always set to "medium" 
+  - Type is always set to "suspicious"
```

Comment:
But since the relay is disputed, it doesn't also fit very much to the Info type, honestly. But the fact about Medium Severity being irrelevant is also correct. Do you think Low Severity and  Suspicious Finding Type would be correct? 

I feel it might be suspicious if a relayer has submitted an invalid root bundle. So it is kind of a "warning." 

The final call is up to you 👍

Reply:
I think I'm convinced! xD